molecule/config.py

```diff
@@ -138,6 +144,10 @@ def action(self, value):
     def project_directory(self):
         return os.getcwd()
 
+    @property
```

Comment:
Ah, this is already actually covered by the `molecule_directory` property. Will remove.

Reply:
Oh wait, got it wrong. `molecule_directory` refers to the role molecule folder, not the cache folder. Ignore me ;)